playground-remote-inference.md

```diff
@@ -0,0 +1,72 @@
+# Remote inference in Playground
+
+Remote inference enables users to inference with external language models in playground.
```

Comment:
"Users can chat with external language models in playground."

Not sure if it's the best feature name, at least for now call it "Remote inference in Playground" to avoid mis-point to the "Deploy ACA remote inference".

BTW, "Add model for remote inference" seems confusing, would e.g., "Add remote inference model to Playground" be better?